scripts/cardano-ledger-specs.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+nix build --accept-flake-config .#fine-types:exe:fine-types
+cd lib/cardano-ledger-specs/
+rm -rf fine-types
+ln -s ../../result/bin/fine-types
+./fine-types check -i Shelley.fine
+./fine-types check -i Allegra.fine
+./fine-types check -i Mary.fine
+./fine-types check -i Alonzo.fine
+./fine-types check -i Babbage.fine
```

Comment:
```suggestion
finetypes=../../result/bin/fine-types
$finetypes check -i Shelley.fine
$finetypes check -i Allegra.fine
$finetypes check -i Mary.fine
$finetypes check -i Alonzo.fine
$finetypes check -i Babbage.fine

```

To avoid doing `rm`? 🤔